### ISSUE TITLE
attach_elb_log_delivery_policy also needed for ALB

### ISSUE THREAD

**Issue by sam0737** [terraform-aws-modules/terraform-aws-s3-bucket#98]
## Description

The document implies that only `attach_lb_log_delivery_policy = true` is needed for ALB logging, but actually `attach_elb_log_delivery_policy = true` is also needed. (NLB would requires `attach_lb_log_delivery_policy = true` only)

### Code Snippet to Reproduce

```
module "s3_logs_lb" {
  source = "terraform-aws-modules/s3-bucket/aws"

  bucket = "test"
  acl    = "log-delivery-write"

  attach_lb_log_delivery_policy = true
}

module "alb" {
  source  = "terraform-aws-modules/alb/aws"
  version = "~> 6.0"
...

  access_logs = {
    bucket = module.s3_logs_lb.s3_bucket_id
    prefix = "alb"
  }
...
}
```

## Expected behavior

Everything works fine.

## Actual behavior

Failed in creating the ALB.

### Terminal Output Screenshot(s)

```
Error: failure configuring LB attributes: InvalidConfigurationRequest: Access Denied for bucket: test. Please check S3bucket permission
...
with module.alb.aws_lb.this[0]
```

## Additional context

According to https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy, `arn:aws:iam::elb-account-id:root` shall be allowed to do `s3:PutObject`.

**Comment by cringdahl** [comment]
Thanks for this, I was starting to tear my hair out.

**Comment by antonbabenko** [comment]
Fixed in 53ac7dd

[v2.6.0](https://github.com/terraform-aws-modules/terraform-aws-s3-bucket/releases/v2.6.0) has been just released.
